db/migrate/20160214115800_inline_ems_id.rb

```diff
@@ -28,7 +28,7 @@ def add_columns
 
   def update_columns
     say_with_time("Inline ems_id in containers") do
-      Container.includes(:container_definitions => :container_group).all.each do |container|
+      Container.includes(:container_definition => :container_group).all.each do |container|
```

Comment:
So there's the above issue that it's loading all the usual model stuff in to `InlineEmsId::Container` *and* I confirmed that in our migrations `includes` can accept anything and doesn't explode:

```ruby
Container.includes(:asdf).all.each { |x| puts x }
# => []
```

While a normal Rails app searches the association as expected:

```ruby
Post.includes(:asdf).all.each { |x| puts x }
# D, [2017-05-11T19:26:03.998481 #5289] DEBUG -- :   Post Load (0.3ms)  SELECT "posts".* FROM "posts"
ActiveRecord::AssociationNotFoundError: Association named 'asdf' was not found on Post; perhaps you misspelled it?
```